### DISCUSSION TITLE
some templates miss when use nuclei with -tags option

### DISCUSSION THREAD

**Discussion by CHYbeta** [projectdiscovery/nuclei#3938] [Q&A]
### Nuclei version:
2.9.8

### Current Behavior:
some templates miss when use nuclei with -tags option

```
cat /root/nuclei-templates/http/vulnerabilities/wordpress/wp-xmlrpc-brute-force.yaml |grep php
  
name: Wordpress XMLRPC.php username and password Bruteforcer
  description: This template bruteforces username and passwords through xmlrpc.php being available.
  tags: wordpress,php,xmlrpc,fuzz
        POST /xmlrpc.php HTTP/1.1

```

however , it will miess when nuclei with tags option
```
nuclei -t /root/nuclei-templates/http/vulnerabilities/wordpress/  -tags php -tl

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v2.9.8

                projectdiscovery.io


Listing available v9.5.7 nuclei templates for /root/nuclei-templates

```

![image](https://github.com/projectdiscovery/nuclei/assets/18642224/38bb303d-81ed-49df-8865-98113076e191)

another example
```
 cat /root/nuclei-templates/http/exposures/backups/php-backup-files.yaml |grep php

id: php-backup-files
  tags: exposure,backup,php,disclosure,fuzz

```

![image](https://github.com/projectdiscovery/nuclei/assets/18642224/59a60adf-3db0-4a27-af55-dcbd923e67ec)

also no result

### Expected Behavior:

should list template with its tags

**Comment by CHYbeta** [comment]
![image](https://github.com/projectdiscovery/nuclei/assets/18642224/7f89336e-c13c-4e5c-896e-8a3a919d430c)


list tpl in /root/nuclei-templates/http/vulnerabilities/wordpress/  with tgs xmlrpc
```
 nuclei -t /root/nuclei-templates/http/vulnerabilities/wordpress/  -tags xmlrpc -tl

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v2.9.8

                projectdiscovery.io


Listing available v9.5.7 nuclei templates for /root/nuclei-templates
http/vulnerabilities/wordpress/wp-xmlrpc-pingback-detection.yaml
```

only http/vulnerabilities/wordpress/wp-xmlrpc-pingback-detection.yaml


however , wp-xmlrpc-brute-force.yaml also tag as `xmlrpc`

https://github.com/projectdiscovery/nuclei-templates/blob/main/http/vulnerabilities/wordpress/wp-xmlrpc-brute-force.yaml


![image](https://github.com/projectdiscovery/nuclei/assets/18642224/b94a427d-adb7-4915-8fd1-5c743d84ceab)

**Comment by ehsandeep** [comment]
@CHYbeta templates with `fuzz` tags are excluded as default, you can include with by adding `-itags fuzz` to include in scan / list.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
